app/views/shared/_footer.html.slim

```diff
@@ -8,8 +8,8 @@ footer.footer.mt-2.pt-2.mb-0
         h5.mt-lg-0.mt-sm-3 #{t('footer.links')}
         ul.m-0.p-0
           li
-            | - 
-            a[href="#"] #{t('footer.link_about_us')}
+            | -
+            a = link_to "#{t('footer.link_about_us')}", about_path
```

Comment:
interpolation
`a = link_to t('footer.link_about_us'), about_path`
or just
`a = link_to t('footer.link_about_us'), about_path`
#link_to method aready should create link